tests/integration/serialization/test_serializers.py

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import random
+
+import pytest
+
+from confluent_kafka import \
+    DoubleSerializer, IntegerSerializer, LongSerializer, \
+    ShortSerializer, StringSerializer, FloatSerializer, AvroSerializer
+
+
+@pytest.mark.parametrize("serializer, data",
+                         [(DoubleSerializer(), random.uniform(-32768.0, 32768.0)),
```

Comment:
Don't use random tests inputs since it makes the tests unreproducible.

Reply:
I  think random number generation is appropriate here. The deserialized content must always match the serialized content so its constant within the scope of the test.